tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -103,7 +103,7 @@ public TracerSettings(IConfigurationSource source)
 #if DEBUG
             ?? 20_000;
 #else
-            ?? 100;
+            ?? 500;
```

Comment:
I noticed in azure functions that higher load scenarios meant the initial requests could take much longer than 100ms and this resulted in a failure and a retry loop on startup. Increasing to 500ms I noticed no timeouts.